Softeq.XToolkit.Bindings.iOS/Gestures/GestureRecognizerBehaviour.cs

```diff
@@ -1,16 +1,17 @@
 // Developed by Softeq Development Corporation
 // http://www.softeq.com
 
-using System.Windows.Input;
+
+using Softeq.XToolkit.Common.Commands;
 using UIKit;
 
 namespace Softeq.XToolkit.Bindings.iOS.Gestures
 {
-    public abstract class GestureRecognizerBehavior
+    public abstract class GestureRecognizerBehaviour<T>
     {
-        public ICommand? Command { get; set; }
+        public ICommand<T>? Command { get; set; }
```

Comment:
**Breaks backward compatibility**.

Usually `Command` uses command from view-model, where we don't have access to UI classes (T= UIPanGestureRecognizer)

Reply:
What do you think about generic versions of Tap<T>...,Bind<T>.This way we make clear that we pass arguments to ICommand.If it's needed to use arguments ,then you need to use generic version,if not-non-generic.This way we make clear that GestureRecognitionBehaviour pass arguments to ICommand .  Now, according to  GestureRecognitionBehaviour interface , it can execute any command and to find out which argument type it passes to command we need to explore TapGestureRecognizerBehaviour.cs,TapGestureRecognizerBehaviour.cs files.

Reply:
The generic version used in < 10% cases, where really need gesture object.

I think that generic methods are not required.